### ISSUE TITLE
Out-of-date packages need updating

### ISSUE THREAD

**Issue by elken** [gatsbyjs/gatsby#16323]
<!--
  Please fill out each section below, otherwise your issue will be closed. This info allows Gatsby maintainers to diagnose (and fix!) your issue as quickly as possible.

  Useful Links:
  - Documentation: https://www.gatsbyjs.org/docs/
  - How to File an Issue: https://www.gatsbyjs.org/contributing/how-to-file-an-issue/

  Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
-->

## Description

A number of packages in base gatsby and other workspaces are out-of-date and open to various levels of exploitation (as highlighted in #16316 and #10583 seems stale)

Updating ts-jest and chokidar in the root workspace resolves those
```diff                                                                                                                                                                                                                                               
diff --git a/package.json b/package.json
index ab94a1333..807176ff6 100644                                                                                                                                                                                                                                               --- a/package.json                                                                                                                                                                                                                                                              
+++ b/package.json                                                                                                                                                                                                                                                              
@@ -7,7 +7,7 @@                                                                                                                                                                                                                                                                      "babel-eslint": "^10.0.1",
     "babel-jest": "^24.0.0",
     "chalk": "^2.4.2",
-    "chokidar": "^1.7.0",                                                                                                                                                                                                                                                      
+    "chokidar": "3.0.2",                                                                                                                                                                                                                                                            "cross-env": "^5.1.4",
     "danger": "^8.0.0",
     "eslint": "^5.16.0",
@@ -39,7 +39,7 @@                                                                                                                                                                                                                                                                    "prettier": "1.17.1",
     "rimraf": "^2.6.1",
     "svgo": "1.2.2",
-    "ts-jest": "^22",                                                                                                                                                                                                                                                          
+    "ts-jest": "24.0.2",                                                                                                                                                                                                                                                            "typescript": "^2.3.2",
     "yargs": "^10.0.3"
   },
```

If we want to force transitive dependencies to be fixed, a `resolutions` field is necessary.

```json
{
   "resolutions": { "**/**/lodash": "^4.17.12" }
}
```

### Steps to reproduce

`npm audit`

### Expected result

Audit should return 0 vulnerable packages

### Actual result

Audit returns non-zero

### Environment

Run `gatsby info --clipboard` in your project directory and paste the output here.

                                                                                                                                                                                                                                                                                 
 System:
    OS: Linux 4.4 Ubuntu 18.04.2 LTS (Bionic Beaver)
    CPU: (4) x64 Intel(R) Core(TM) i5-7400 CPU @ 3.00GHz
    Shell: 5.4.2 - /bin/zsh
  Binaries:
    Node: 12.7.0 - /usr/bin/node
    Yarn: 1.17.3 - /usr/bin/yarn
    npm: 6.10.0 - /usr/bin/npm
  Languages:
    Python: 2.7.15+ - /usr/bin/python

**Comment by m-allanson** [comment]
Chokidar in particular would be good to get updated. The v3 changelog says:

> Massive CPU & RAM consumption improvements. 17x package & deps size reduction. Node 8+-only

**Comment by elken** [comment]
That's it for the most part. The biggest ones are the Lodash dependencies, which we can enforce via `npm-shrinkwrap` if pdf2json doesn't update.

**Comment by lannonbr** [comment]
Resolved in #16767. Closing
